.github/workflows/backend-api-tests.yml

```
@@ -0,0 +1,72 @@
+name: API Tests
+
+on:
+  pull_request:
+    branches: [main, develop]
+    types: [opened, synchronize]
+    paths:
+      - "backend/**"
+      - ".github/workflows/backend-api-tests.yml"
+    workflow_dispatch:
```

actionlint (1.7.8)

unexpected key "workflow_dispatch" for "pull_request" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)